src/styles/main.scss

```diff
@@ -0,0 +1,35 @@
+.main {
+  --column-count: 4;
+
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 1fr);
+  justify-content: center;
+  gap: 46px 48px;
+  max-width: fit-content;
+  margin: 0 auto;
+}
+
+@media (min-width: 300px) {
+  .main {
+    --column-count: 1;
+  }
+}
+
+@media (min-width: 488) {
```

Comment:
The `min-width` value should include a unit, such as `px`. Change `min-width: 488` to `min-width: 488px` to ensure the media query functions correctly.